lib/elastic/enterprise-search/request.rb

```diff
@@ -50,7 +50,7 @@ def request(method, path, params = {})
             # is Charles, which uses a self-signed certificate in order to inspect https traffic. This will
             # not be part of this client's public API, this is more of a development enablement option
             http.verify_mode = ENV['st_ssl_verify_none'] == 'true' ? OpenSSL::SSL::VERIFY_NONE : OpenSSL::SSL::VERIFY_PEER
-            http.ca_file = File.join(File.dirname(__FILE__), '..', 'data', 'ca-bundle.crt')
+            http.ca_file = File.join(File.dirname(__FILE__), '../..', 'data', 'ca-bundle.crt')
```

Comment:
Might want to make this another argument to `join` instead: `File.join(File.dirname(__FILE__), '..', '..', 'data', 'ca-bundle.crt')`

You could also use something like `File.realpath` to make sure the file actually exists